.circleci/config.yml

```diff
@@ -271,41 +271,10 @@ workflows:
   version: 2
   test:
     jobs:
-      - core-go-test:
```

Comment:
Should we remove the actual job specs too? Not just their references

Reply:
but I guess they're in source control